### PR TITLE
Fix for new update

### PR DESCRIPTION
**Pull request by developer9998**
-Fixed Forest computer mesh location
-Adjusted Visual and Color/Colour views
-Fixed the group tab (I think)
-More changes but that's all the important ones

### CODE REVIEW

File: ComputerInterface/Views/GameSettings/GroupView.cs [hunk 1]
```diff
@@ -56,11 +57,10 @@ public void DrawHeader(StringBuilder str)
...
         {
-            str.Append(_selectionHandler.GetIndicatedText(0, "Forest")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(1, "Cave")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(2, "Canyon")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(3, "City")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(4, "Mountain")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(0, "Forest  ")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(1, "Cave    ")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(2, "Canyon  ")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(3, "City    ")).AppendLine();
```
Comment (Graicc) [inline ComputerInterface/Views/GameSettings/GroupView.cs]: What happened to mountain?

Comment (developer9998) [reply ComputerInterface/Views/GameSettings/GroupView.cs]: i couldn't get it working.

Comment (developer9998) [reply ComputerInterface/Views/GameSettings/GroupView.cs]: all of them worked fine but when i tried mountain it didnt work + none worked in mountain

Comment (developer9998) [reply ComputerInterface/Views/GameSettings/GroupView.cs]: ill prob try to add it back when i have the time..

Comment (Graicc) [reply ComputerInterface/Views/GameSettings/GroupView.cs]: Fixed in commit 7b314ddc3f71260aa748c6729ec6955d1d22870c

**Pull request merged**